qcengine/programs/gamess/harvester.py

```diff
@@ -125,6 +145,21 @@ def harvest_outfile_pass(outtext):
             qcvar['CCSD CORRELATION ENERGY'] = mobj.group(3)
             qcvar['CCSD TOTAL ENERGY'] = mobj.group(2)
 
+        # Process CR-CC(2,3)
+        mobj = re.search(
+            r'^\s+' + r'CCSD                       ENERGY:'       + r'\s+' + NUMBER + r'\s*' + r'CORR.E=\s+' + r'\s+' + NUMBER + r'\s*' +
+            r'^\s+' + r'CR-CC\(2,3\),A OR CCSD\(2\)_T  ENERGY:'   + r'\s+' + NUMBER + r'\s*' + r'CORR.E=\s+' + r'\s+' + NUMBER + r'\s*' +
+            r'^\s+' + r'CR-CC\(2,3\) OR CR-CCSD\(T\)_L ENERGY:'   + r'\s+' + NUMBER + r'\s*' + r'CORR.E=\s+' + r'\s+' + NUMBER + r'\s*'
```

Comment:
will have to watch out later when CCSD(2)_T or CR-CCSD(T)_L requested that they don't get the same labels.